libminifi/include/core/state/nodes/AgentInformation.h

```diff
@@ -643,16 +704,7 @@ class AgentInformation : public DeviceInformation, public AgentMonitor, public A
   }
 
   std::vector<SerializedResponseNode> serialize() {
-    std::vector<SerializedResponseNode> serialized;
-
-    SerializedResponseNode ident;
-
-    ident.name = "identifier";
-    ident.value = identifier_;
-
-    SerializedResponseNode agentClass;
-    agentClass.name = "agentClass";
-    agentClass.value = agent_class_;
+    std::vector<SerializedResponseNode> serialized(std::move(AgentInformationWithoutManifest::serialize()));
```

Comment:
`std::move` is redundant, since the result of a function call expression is already an rvalue. It will prevent copy/move elision, thus making the end result even slower than without `std::move`.

Reply:
good point. Will remove it